lm75bd/lm75bd.c

```diff
@@ -26,15 +31,40 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
-error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
+error_code_t readTempLM75BD(uint8_t devAddr, float *temp)
+{
+
+  if (temp == NULL)
+    return ERR_CODE_INVALID_ARG;
+
+  uint8_t buffer[TEMP_REG_SIZE_B] = {0};
+  uint8_t tempBuf = LM75BD_REG_TEMP;
+
+  error_code_t sendErr = i2cSendTo(devAddr, &tempBuf, 1);
+  if (sendErr != ERR_CODE_SUCCESS)
+  {
+    return sendErr;
+  }
+
+  error_code_t readErr = i2cReceiveFrom(devAddr, &buffer, 2);
+  if (readErr != ERR_CODE_SUCCESS)
+  {
+    return readErr;
+  }
```

Comment:
you can just use the same error code for both of these instead of creating a new error code variable each time